test/extended/builds/s2i_quota.go

```diff
@@ -53,11 +53,9 @@ var _ = g.Describe("[Feature:Builds][Conformance] s2i build with a quota", func(
 				o.Expect(br.Build.Status.Duration).To(o.Equal(duration), "Build duration should be computed correctly")
 
 				g.By("expecting the build logs to contain the correct cgroups values")
-				// buildLog, err := br.LogsNoTimestamp()
-				_, err = br.LogsNoTimestamp()
+				buildLog, err := br.LogsNoTimestamp()
 				o.Expect(err).NotTo(o.HaveOccurred())
-				// TODO: re-enable this check when https://bugzilla.redhat.com/show_bug.cgi?id=1764323 is resolved
-				//o.Expect(buildLog).To(o.ContainSubstring("MEMORY=419430400"))
+				o.Expect(buildLog).To(o.ContainSubstring("MEMORY=419430400"))
 				// TODO: re-enable this check when https://github.com/containers/buildah/issues/1213 is resolved.
 				//o.Expect(buildLog).To(o.ContainSubstring("MEMORYSWAP=419430400"))
```

Comment:
@nalind can we re-enable this check, too?

Reply:
@nalind bump..I saw some discussion in the bugs around this but i didn't follow it closely enough to understand if the memoryswap values should be correct now or not.

Reply:
I don't expect it to be set to the value that the test would expect, since cri-o can't be told to do it for us (there doesn't appear to be a `k8s.io/api/core/v1.ResourceName` for swap limits - it can be told to set memory usage, and https://github.com/cri-o/cri-o/pull/2997 fixed a problem with it, allowing us to re-enable the `MEMORY=` test), and we don't currently have logic in buildah's chroot isolation that would attempt to set control group settings.

Reply:
@nalind if that's the case, can you add a commend to this effect in https://github.com/containers/buildah/issues/1213?

Reply:
Added that [note](https://github.com/containers/buildah/issues/1213#issuecomment-571277448).  I'm _really_ hoping we can lean on libcontainer for that.